frontend/src/components/organization/MiniOrganizationPreview.js

```diff
@@ -72,15 +73,16 @@ export default function MiniOrganizationPreview({
           size={size}
           onDelete={onDelete}
           doNotShowName={doNotShowName}
```

Comment:
Out of scope for this PR, but do you know why we have a mix of the negative like `doNotShow` and positive like `show` prefixes? I think from a component API perspective we should align on one pattern.

Reply:
I had defined doNotShowName as a negative because there was only 1 instance of `MiniOrganizationPreview` where I didn't want the name to be shown and I didn't want to modify the props of the other instances everywhere else.